blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/ElasticsearchRestHelper.java

```diff
@@ -347,4 +349,22 @@ private String getBoolQueryString(String mustValueString, String shouldValueStri
     public void setBaseUrlForTestOnly(String url) {
         baseUrl = url;
     }
+
+    @VisibleForTesting
+    public int refreshIndex(String indexName) throws IOException {
```

Comment:
Yay, so this worked for you I assume?

Reply:
Yes. It was good to switch out of hard-coded sleep. I still see many legacy *sleep* code in there. I think when we are touching that piece of functionality, we will take care of them at that time.